src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArr = sourceString.split(';');
+  const obj = {};
+
+  const arr = sourceArr.map(elem => elem.split(':').map(item => item.trim()));
+
+  arr.map((item) => {
```

Comment:
```suggestion
  arr.forEach(([key, value]) => {
```
Используй forEach вместо map, ты никак не используешь результат map
И можно сразу деструктуризировать нужные значения из массива, который в item